website/docs/language/meta-arguments/lifecycle.mdx

```diff
@@ -49,6 +49,14 @@ The arguments available within a `lifecycle` block are `create_before_destroy`,
   such features, so you must understand the constraints for each resource
   type before using `create_before_destroy` with it.
 
+  It's important to understand that `create_before_destroy` meta-attribute is
+  stored in state file and _propagated_ to all resource dependencies, including
```

Comment:
Note for docs team reviewer: "propagated" may not be the correct term to use here.

Reply:
Whatever reflects the fact of the meta-attribute proliferation through the resource graph 🙂

Reply:
Indeed! That was feedback from triage today, just wanted to capture it for the future.